src/styles/theming/sap_fiori_3.scss

```diff
@@ -175,6 +175,7 @@
   --fdMessage_Box_Shadow: var(--sapContent_Shadow1);
   --fdMessage_Border: none;
   --fdMessage_Border_Radius: 0;
+  --fdMessage_Icon_Display: none;
```

Comment:
These values should go in the common folder/file as they are the same across Quartz themes and Horizon themes